packages/vite/src/node/cli.ts

```diff
@@ -124,6 +124,7 @@ cli
       `or specify minifier to use (default: terser)`
   )
   .option('--manifest', `[boolean] emit build manifest json`)
+  .option('--watch', `[boolean] rebuilds when modules have changed on disk.`)
```

Comment:
**nitpick (non-blocking):** All other descriptions don't end with a period, so for a better consistency we should remove it :slightly_smiling_face:  

```suggestion
  .option('--watch', `[boolean] rebuilds when modules have changed on disk`)
```